README.md

```diff
@@ -265,7 +265,7 @@ Note: By default, `Auto-scroll` is checked and new entries in the log will autom
 
 To benefit from Java Language completion on standalone files, [VS Code Language support for java](https://marketplace.visualstudio.com/items?itemName=redhat.java) needs to be installed. An invisible project is created with a default classpath.
 
-The command `Refresh local Java classpath for Camel K standalone file based on current editor. Available with kamel 1.4+.` allows to have specfic dependencies of the opened Integration part of the classpath. For isntance, specific dependencies can be decalred as modeline.
+The command `Refresh local Java classpath for Camel K standalone file based on current editor. Available with kamel 1.4+.` allows to have specific dependencies of the opened Integration part of the classpath. For instance, specific dependencies can be declared as modeline. When a Camel K modeline is provided, a CodeLens is provided to refresh the classpath dependencies.
```

Comment:
This seems a bit awkward as a paragraph to me. I doubt that many users will know what a CodeLens is. It may be useful to provide a link to https://code.visualstudio.com/blogs/2017/02/12/code-lens-roundup or http://theblogreaders.com/visual-studio-code-tips-codelens-visual-studio-code/ or some other source (not a lot of info on CodeLens from a user perspective in the VS Code site beyond API references). 

Also I might suggest a few tweaks to the language here...

"The command X refreshes specific dependencies for the classpath of the opened Integration, such as dependencies declared as part of the modeline of the file. When using a modeline to configure such dependencies, you gain a CodeLens link ("Refresh classpath dependencies") at the top of the editor to trigger the refresh more easily."